app/components/DoubtsSection/style.module.css

```diff
@@ -0,0 +1,61 @@
+.doubts__section {
+  padding-top: 4.5rem;
+  padding-bottom: 8rem;
+
+  text-align: center;
+}
+
+.doubts__title {
+  margin-bottom: 1rem;
+}
+
+.doubts__description {
+  margin-bottom: 4.5rem;
+}
+
+.tabs__box {
+  width: 100%;
+  
+  margin-bottom: 1rem;
+
+  border-bottom: 1px solid var(--neutral-gray-medium);
+}
+
+.tabs__box>div>div{
+  overflow-x: scroll;
+}
+
+.tab__box{
+  max-height: min-content;
+  max-width: 33% !important;
```

Comment:
Fiz um teste tirando esse max-width e botando 
width: fit-content;
flex-shrink: 1!important;

E gostei. Aproximou do Figma. Problema: suporte no Firefox nao eh bom para fit-content. Talvez deixa um width 33% como fallback

Reply:
Só para ver se eu entendi, deixar como está e adicionar o width fit-content?
Acho até que eu tinha no início colocado width: fit-content, mas removi por que ele não era o suficiente para quebrar o button da categoria.

Reply:
O que eu fiz foi tirar o max-width
Adicionar width 33%
depois dele coloca width fit-content
e o flex-shrink: 1

Reply:
@marcelluscaio , ao remover o max-width: 33% e substituir por width: 33% só, ele perde o layout esperado para a versão desktop.

Reply:
Ta certissima, nao tinha me tocado disso!